loolwsd-systemplate-setup

```diff
@@ -105,23 +105,33 @@ done
 
 cd $CHROOT || exit 1
 
+# Copy fonts directories, recursively, with filters.
+# copy_fonts_dir <source_dir> <target_dir>
+function copy_fonts_dir
+{
+    # All the font files are cached in Core at pre-init
+    # from the system, so we don't need them in the jail.
+    # Not having the .ttf files gives us transparent text!
+    find $1 -not -iname "*.ttc" \
+            -not -iname "*.pcf" -not -iname "*.pcf.gz" \
+            -not -iname "*.bdf" -not -iname "*.otf" \
```

Comment:
This leaves out all but TTF files from *systemplate*. For some reason removing .ttf files from systemplate results in transparent text.

This does *not* mean .ttf files are copied in the jails (see Kit.cpp patch).